common/src/main/scala/tracking/DynamoTopicSubscriptionsRepository.scala

```diff
@@ -13,14 +14,14 @@ import scala.collection.JavaConversions._
 
 class DynamoTopicSubscriptionsRepository(client: AsyncDynamo, tableName: String)
   (implicit ec: ExecutionContext) extends TopicSubscriptionsRepository {
-
+  val logger = Logger(classOf[DynamoTopicSubscriptionsRepository])
   val TopicIdField = "topicId"
   val TopicSubscriberCountField = "topicSubscriberCount"
 
-  override def subscribe(topic: Topic): Future[RepositoryResult[Unit]] =
+  override def deviceSubscribed(topic: Topic): Future[RepositoryResult[Unit]] =
```

Comment:
Naming was wrong in that sense `TopicSubscriptionsRepository` is not responsible for subscribing to topics (as methods suggested). I wanted to emphasise fact it is `notified` on topic subscription instead.